source/fundamentals/crud/compound-operations.txt

```diff
@@ -48,10 +48,14 @@ configurable :ref:`sort <node-fundamentals-sort>` and
 
    Starting in version 5.7, you can set the ``includeResultMetadata``
    setting in the ``options`` object to specify the return type for each
-   of these methods. The setting defaults to ``true``, which means the
-   method returns a ``ModifyResult`` type. If you set
-   ``includeResultMetadata`` to ``false``, the method returns the
-   modified document.
+   of these methods.
+   
+   Starting in version 6.0, the setting defaults to
```

Comment:
Q: This might be a larger team discussion, but I'm always unsure about whether to include "Starting in..." language like this, since the change will only be published in that version anyway. Do you think it's helpful to users?

Reply:
Not making any changes, as we should have a team discussion- will address in a cleanup ticket later depending on discussion outcome